Update Bundle and NPM Dependencies

CDAGaming opened the pull request:
Listen 3.1.2+ Incompatible at this time, not updated

Signed-off-by: CDAGaming <cstack2011@yahoo.com>

CDAGaming committed:
Update Bundle and NPM Dependencies

Listen 3.1.2+ Incompatible at this time, not updated

Signed-off-by: CDAGaming <cstack2011@yahoo.com>

CDAGaming commented:
Already Outdated Base, Geez this updates quickly 👍

CDAGaming committed:
Merge branch 'v4-dev' into v4-dev

CDAGaming committed:
Merge branch 'v4-dev' into v4-dev

CDAGaming committed:
Merge remote-tracking branch 'origin/v4-dev' into v4-dev

CDAGaming committed:
Merge branch 'v4-dev' of https://github.com/CDAGaming/bootstrap into v4-dev

CDAGaming committed:
Merge branch 'v4-dev' into v4-dev

CDAGaming committed:
Merge branch 'v4-dev' into v4-dev

CDAGaming committed:
Updates More Dependencies

CDAGaming committed:
Merge branch 'v4-dev' into v4-dev

Johann-S commented on package.json:
```diff
@@ -86,12 +86,13 @@
+    "shelljs": "^0.7.8",
     "shx": "^0.2.2",
-    "uglify-js": "^3.0.13"
+    "uglify-js": "^3.0.18"
```
Strange the last release of uglify-js is 3.0.24 🤔

CDAGaming replied:
Fixed, been on vacation so sorry for downtime xD

patrickhlauke committed:
Remove ::first-letter and ::first-line selectors from print styles

Bootstrap itself doesn't use any `::first-letter` or `::first-line`
styles. These selectors also cause(d) problems in IE11 (see
https://github.com/h5bp/html5-boilerplate/pull/1799) and currently cause
a nasty rendering bug in Chrome where the first letter is vertically
shifted (see https://github.com/twbs/bootstrap/issues/21771)

As h5bp has now removed these as well, no need to keep and comment them
out. See https://github.com/h5bp/html5-boilerplate/issues/1961 /
https://github.com/h5bp/html5-boilerplate/pull/1962

CDAGaming committed:
Updates Dependencies

Signed-off-by: CDAGaming <cstack2011@yahoo.com>

CDAGaming committed:
Merge branch 'v4-dev' of https://github.com/CDAGaming/bootstrap into v4-dev

Signed-off-by: CDAGaming <cstack2011@yahoo.com>

# Conflicts:
#	package.json
